### PR TITLE
GUACAMOLE-30: Implement systemd startup for guacd

### PR DESCRIPTION
**Pull request by necouchman**

### CODE REVIEW

File: src/guacd/systemd/guacd.service.in [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Unit]
```
Comment (mike-jumper) [inline src/guacd/systemd/guacd.service.in]: Do systemd unit files support comments? Assuming they do, this new file should include the usual license boilerplate.

File: src/guacd/systemd/guacd.service.in [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+Description=Guacamole Server
+Documentation=man:guacd(8)
+
+[Service]
+User=daemon
+ExecStart=@sbindir@/guacd -f
```
Comment (mike-jumper) [inline src/guacd/systemd/guacd.service.in]: Is this the generally-accepted best practice? Relying on systemd to handle the daemonization for the process, management of any PID file, etc.?

Comment (necouchman) [reply src/guacd/systemd/guacd.service.in]: Yes - most of what I've seen in systemd unit files is that you put the process in foreground mode and allow systemd to manage it.  I can't cite a source that says it, just my anecdotal experience.

File: src/guacd/systemd/guacd.service.in (hunk 1 above)
Comment (necouchman) [reply src/guacd/systemd/guacd.service.in]: Probably so...will add it.

Comment (necouchman) [reply src/guacd/systemd/guacd.service.in]: Added.

File: src/guacd/systemd/guacd.service.in (hunk 2 above)
Comment (mike-jumper) [reply src/guacd/systemd/guacd.service.in]: Googling around a bit, definitely seems the way of things for systemd: http://www.mikeperham.com/2014/09/22/dont-daemonize-your-daemons/

**Pull request merged**